spellbook/models/opensea/solana/opensea_solana_transfers.sql

```diff
@@ -0,0 +1,136 @@
+{{ config(
+        alias ='transfers',
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key='unique_trade_id'
+        )
+}}
+
+with iv_raw as (
+    select block_time
+          ,block_slot
+          ,id
+          ,case when array_contains(log_messages, 'Program log: Instruction: Sell') then 'Offer Accepted'
+                when array_contains(log_messages, 'Program log: Instruction: Buy') then 'Buy Now'
+                else 'Other'
+           end purchase_method
+          ,case when array_contains(log_messages, 'Program log: Instruction: Sell') then instructions[6]
+                when array_contains(log_messages, 'Program log: Instruction: Buy') then instructions[7]
+                else instructions[7]
+           end inst
+      from {{ source('solana','transactions') }}
+      where 1=1
+      and success
+      {% if is_incremental() %}
```

Comment:
fyi, i updated the incremental logic. using this approach vs. now() allows more flexibility if there are errors and data falls behind more than two days